rotkehlchen/tests/api/test_misc.py

```diff
@@ -315,6 +315,22 @@ def test_manage_nodes(rotkehlchen_api_server):
             'active': True,
         },
     )
+    result = assert_error_response(
+        response=response,
+        contained_in_msg='Node for ethereum with endpoint ewarwae already exists in db',
```

Comment:
What is ewarwae?

Reply:
the fake endpoint that is used in the tests